board/embeddedartists/mx7ulpea-ucom/mx7ulpea-ucom.c

```diff
@@ -13,11 +13,25 @@
 #include <asm/arch/iomux.h>
 #include <asm/mach-imx/boot_mode.h>
 #include <asm/gpio.h>
+#include <fs.h>
 #include <usb.h>
 #include <dm.h>
+#include <spi_flash.h>
```

Comment:
Should we have these protected with? `#ifdef CONFIG_FPGA` -- I know we're enabling the CONFIG in `include/configs/mx7ulpea-ucom.h` but we do an `#ifdef` check below in the source.

Reply:
we can follow that path (@ricardosalveti  did that with the SPL include) but IMO it makes the code a bit uglier and for not much gain...anyhow let me know if you think you'd better see that for consistency

Reply:
Yeah, seems the standard in u-boot is to just include the header when it is building for SPL, so probably better to protect here as well.